introduction-to-bedrock/bedrock_mistral.py

```diff
@@ -0,0 +1,100 @@
+import boto3
+import json
+
+#Create the connection to Bedrock
+bedrock = boto3.client(
+    service_name='bedrock',
+    region_name='us-west-2', 
```

Comment:
We should add a comment here saying the mistral is available on us-west-2 only for now

Reply:
fixed